### DISCUSSION TITLE
Drag and drop note onto calendar to upgrade to event

### DISCUSSION THREAD

**Discussion by Elyassine94** [davish/obsidian-full-calendar#351] [Feature Requests]
Would be nice if we can directly drag and drop a note to the calendar.
Maybe we can even make the plugin directly create the frontmatter fields necessary or detect existing ones.

**Comment by davish** [comment]
Thanks for the suggestion! There's already functionality for "upgrading" a note and adding frontmatter, and [FullCalendar natively supports dragging DOM elements onto the calendar](https://fullcalendar.io/docs/external-dragging). So it should be possible to support dragging files from the file explorer on to an open calendar.

**Reply by jlc467** [reply]
Hi,
Love this plugin! I read the docs and I'm not seeing the functionality for "upgrading" a note and adding frontmatter. I'm imagining some command that takes the selected plain-old-obsidian-note and upgrades it to have a date in the title and the frontmatter necessary to show on calendar. Of course, dragging and dropping from the file explorer would be even better -- if that's not being worked on, I might take a stab at it. Otherwise, maybe just a suite of frontmatter "upgrade" commands for e.g. "today", "tomorrow", "next week" that I can assign to hot keys.
